client/src/commons/Forms/PlacesTypesAndSubTypes/PlacesTypesAndSubTypes.tsx

```diff
@@ -94,19 +94,22 @@ const PlacesTypesAndSubTypes: React.FC<PlacesTypesAndSubTypesProps> = (props: Pl
                                 control={control}
                                 render={(props) => (
                                     <Autocomplete
+                                        defaultValue=''
```

Comment:
Why is the default value blank here?

Reply:
because I need some value to be present when the page first renders so the x will appear and all the values in this autocomplete are fetched so I can't use them.